docusaurus/docs/Android/03-ui/06-guides/08-adding-custom-attachments.mdx

```diff
@@ -18,10 +21,11 @@ You can find the full code from this guide on [GitHub](https://github.com/GetStr
 
 ## Sending Date Attachments
 
-First of all, you'll need to customize the leading content of [MessageComposerView](../04-message-components/04-message-composer.mdx) to let the user pick a date using the date picker button. We'll take the [DefaultMessageComposerLeadingContent](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/composer/content/DefaultMessageComposerLeadingContent.kt) as a reference and add a new date picker button:
+First of all, you'll need to customize the leading content of [MessageComposerView](../04-message-components/04-message-composer.mdx) to let the user pick a date using the date picker button. We'll take the [DefaultMessageComposerLeadingContent](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/feature/messages/composer/content/DefaultMessageComposerLeadingContent.kt) as a reference and add a new date picker button:
 
 ```xml
 <merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
```

Comment:
We haven't added a name for this layout above it.
Since we're binding in the examples, it's important to have an exact name or the users could get frustrated.